tests/gnrc_sock_dodtls/README.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS over DTLS
+client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+```console
+$ sudo ip tuntap add dev tap0 mode tap user $(id -u -n)
+$ sudo ip a a 2001:db8::1/64 dev tap0
+$ sudo ip link set up dev tap0
+```
+
+TBD: add how to start DNS over DTLS server here
```

Comment:
Indeed - what DNS over DTLS server do you recommend for this?
Are there public ones?

Reply:
See testing procedures in OP. Will add them to the doc.

Reply:
There is also the Go-based [`routedns`](https://github.com/folbricht/routedns) which also supports DoDTLS, but I did not try that one yet. Given the finickiness of TinyDTLS, I don't expect much success either, but maybe it works with the `TLS_PSK_WITH_AES_128_CCM_8` ciphersuite.